src/Style.hs

```diff
@@ -283,12 +269,40 @@ attribute = identifier -- TODO: Naming convention - same as identifiers?
 -- styPrettyPrint b = ""
 
 --------------------------------------------------------------------------------
--- Functions used by "Runtime"
+-- Functions used by "Runtime" -- Style Translator
+-- Translates from a Style AST to:
+-- * a list of geometric primitives
+-- * a list of constraints
+-- * a list of objectives
+
+-- Type aliases for readability in this section
+-- | 'StyContext' maintains the current output of the translater
+type StyContext a =
```

Comment:
this name might be confusing because i refer to a VarMap as a context (and in PL a context is typically a list of `varName : varType` of variables in the current scope)

Reply:
I think things similar to `VarMap` are usually called symbol table. I always thought "context" is a set of environment variables that includes a symbol table. How about `StyEnv` for "Style environment"?

Reply:
i agree that symbol table is a better name for `VarMap`, but not sure about environment, since this type describes something that's more of a "concrete output" and not just "auxiliary information"